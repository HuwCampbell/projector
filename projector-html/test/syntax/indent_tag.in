case x of
  y ->
    <a b="{{ x
      }}">
    </a>
