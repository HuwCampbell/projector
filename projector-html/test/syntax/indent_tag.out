Template
  ()
  Nothing
  (TECase
     ()
     (TEVar () TId { unTId = "x" })
     (TAlt
        ()
        (TPVar () TId { unTId = "y" })
        (TENode
           ()
           (THtml
              ()
              [ TElement
                  ()
                  (TTag () "a")
                  [ TAttribute
                      ()
                      TAttrName { unTAttrName = "b" }
                      (TQuotedAttrValue
                         () (TIString () [ TExprChunk () (TEVar () TId { unTId = "x" }) ]))
                  ]
                  (THtml () [ TWhiteSpace () 1 ])
              ])) :|
        []))